core/src/ldml/C7043_ldml.md

```diff
@@ -324,10 +324,87 @@ For each key:
 - `vkey`: Is the standard vkey, 0-255
 - `target`: Is the target (resolved) vkey, 0-255.
 
+### C7043.2.13 `layr`—Layers list
+
+Represents layers on the keyboard.
+- Each key entry corresponds to a key on the row
+
+| ∆ | Bits | Name       | Description                              |
+|---|------|------------|------------------------------------------|
+| 0 |  32  | ident      | `layr`                                   |
+| 4 |  32  | size       | int: Length of section                   |
+| 8 |  32  | listCount  | int: Number of layer lists               |
+|12 |  32  | layerCount | int: number of layer entries             |
+|16 |  32  | rowCount   | int: number of row entries               |
+|20 |  32  | keyCount   | int: number of key entries               |
+|24 |  64  | reserved   | padding                                  |
+
```

Comment:
Suggest listing the variable length data here in the table, and then adding subheadings for each sub-table, e.g. `#### layers sub-table`, `#### rows sub-table`, `#### keys sub-table`
```suggestion
|32 | var  | layers     | layers sub-table                         |
| - | var  | rows       | rows sub-table                           |
| - | var  | keys       | keys sub-table                           |

```

Reply:
this is a great suggestion that gives the overall picture in the main table